godtools/ViewControllers/Platform/LanguagesTableViewController.swift

```diff
@@ -57,9 +64,17 @@ class LanguagesTableViewController: BaseViewController {
     
     // MARK: - Helpers
     
+    func primaryLanguage(primary:Bool) {
```

Comment:
A couple things here:
1. Could you change the method name to `selectingPrimaryLanguage(..)`. It's current name makes me think that I would be passing a Language in, not a bool.
2. Using the underscore in the method signature (before "primary") prevents the caller from having to type "primary"